tests/integration/api_v3/http/transactionsDryRun.test.js

```diff
@@ -48,9 +49,8 @@ describe('Post dryrun transactions API', () => {
 		);
 
 		expect(response).toMap(goodRequestSchema);
-		expect(response.data).toMap(dryrunTransactionResponseSchema);
+		expect(response.data).toMap(dryrunTransactionSuccessResponseSchema);
```

Comment:
Please update all the test descriptions (wherever necessary) to be more accurate, explaining what is being tested.